README.md

```diff
@@ -71,6 +71,16 @@ More information on the ConflictMonitor and other services described above can b
 **Ongoing Efforts**
 This feature is under active development. This is a joint effort involving combining the features of the existing CIMMS conflictvisualizer tools with the CVManager components, to enable connected vehicle and intersection analysis in one application.
 
+### Feature Flags
+
+This application has the ability to disable certain features based on environment variables. For each of these variables, the feature will be enabled if the variable is anything but 'false'. These features include:
+
+- ENABLE_RSU_FEATURES: if 'false', disable all RSU-specific features, including map, RSU data, RSU configuration, and RSU organization linking.
+- ENABLE_INTERSECTION_FEATURES: if 'false', disable all intersection-specific features, including WZDx data and intersection map

```

Comment:
I'm not sure these are describing accurately what should happen. intersection/wzdx are the same description?

Reply:
updated